open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/open-lineage-janus-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/openlineageconnectors/janusconnector/factory/GraphFactory.java

```diff
@@ -119,21 +117,20 @@ private JanusGraph initializeGraph(JanusGraph janusGraph, String graphType) thro
     /**
      * Set up the labels of the schema for the Janus Graph instance
      *
-     * @param labels  - set of labels
-     * @param management  - management instance of Janus Graph
-     * @param type  - type to be labeled
-     *
+     * @param labels     - set of labels
+     * @param management - management instance of Janus Graph
+     * @param type       - type to be labeled
      */
-    private JanusGraphManagement checkAndAddLabelVertexOrEdge(Set<String> labels, JanusGraphManagement management,String type){
-        if(type.equals("vertex")){
-            for (String label: labels) {
+    private JanusGraphManagement checkAndAddLabelVertexOrEdge(Set<String> labels, JanusGraphManagement management, String type) {
```

Comment:
This can be separated into different methods, with diminished parameters and specific responsibilities. Also, do not understand why management is returned, void can used. Streams can be used, even this method can be removed entirely.